### DISCUSSION TITLE
Embedded lyrics - Lyrics not available

### DISCUSSION THREAD

**Discussion by Sagnasty** [rern/rAudio#1145] [Fixed Issues] [answered]
Running rAudio 20221210.

When trying to display lyrics for a track, a message is displayed (Lyrics not available.)

I confirmed that the lyrics are indeed embedded within the file (flac).

Jim

3 x Raspberry Pi 3B+ Rev 1.3 > HiFiBerry DAC+ Pro
1 x Raspberry Pi 3B Rev 1.2 > HiFiBerry DAC+ Pro
Synology DS718+

**Comment by rern** [comment, marked as answer]
Hi Jim,

Try: Features > Lyrics in File - enable

**Reply by Sagnasty** [reply]
Hi Rern,

That solved my problem - thanks

Jim

3 x Raspberry Pi 3B+ Rev 1.3 > HiFiBerry DAC+ Pro
1 x Raspberry Pi 3B Rev 1.2 > HiFiBerry DAC+ Pro
Synology DS718+
